visualization/include/pcl/visualization/impl/point_cloud_color_handlers.hpp

```diff
@@ -73,6 +73,47 @@ pcl::visualization::PointCloudColorHandlerCustom<PointT>::getColor (vtkSmartPoin
 }
 
 ///////////////////////////////////////////////////////////////////////////////////////////
+template <typename PointT> void
+pcl::visualization::PointCloudColorHandlerCustomIndividual<PointT>::setInputCloud (
+    const PointCloudConstPtr &cloud)
+{
+  PointCloudColorHandler<PointT>::setInputCloud (cloud);
+  
+  if (cloud_->points.size () == rgb_values_.size())
+    capable_ = true;
+  else
+    capable_ = false;
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////
+template <typename PointT> bool
+pcl::visualization::PointCloudColorHandlerCustomIndividual<PointT>::getColor (vtkSmartPointer<vtkDataArray> &scalars) const
+{
+  if (!capable_ || !cloud_)
```

Comment:
Bool cast for shared pointers [returns](http://www.boost.org/doc/libs/1_61_0/libs/smart_ptr/shared_ptr.htm#conversions) `get() != 0`.